eng/BootStrapMsBuild.targets

```diff
@@ -63,14 +63,35 @@
     </ItemGroup>
   </Target>
 
+  <!-- The task allows to find VS bits on machine. These files will be used as a source for patching on the top of them. -->
```

Comment:
is there a reason this had to be in this PR?

Reply:
otherwise, the e2e tests are failing on windows core run